src/client/components/home/MealList.js

```diff
@@ -0,0 +1,44 @@
+import React, { createContext, useEffect, useState } from "react";
+import Meal from "./Meal";
+import "./meals.css";
+export const MealContext = createContext();
+function MealList() {
+  const [meals, setMeals] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    getResult();
+  }, []);
+  const getResult = async () => {
+    setLoading(true);
+    try {
+      const response = await fetch("/api/meals");
+      const result = await response.json();
+      console.log(result);
+      setMeals(result);
+    } catch (err) {
+      setError(err);
+    }
+    setLoading(false);
+  };
+
+  const results = meals.map((meal) => <Meal key={meal.id} meal={meal} />);
```

Comment:
I'd name this just `meals` or `mealItems`, at this point we no longer care that it's the result from an API call, it's just state we are using.